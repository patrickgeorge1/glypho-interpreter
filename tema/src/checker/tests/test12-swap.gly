MMMJHHH9||^^rr))nxxxrFFF