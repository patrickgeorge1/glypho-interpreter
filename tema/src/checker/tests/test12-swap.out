1321848
871234115
